src/wallet/db.cpp

```diff
@@ -554,7 +554,6 @@ bool CDB::Rewrite(CWalletDBWrapper& dbw, const char* pszSkip)
         }
         MilliSleep(100);
     }
-    return false;
```

Comment:
Mixed feelings here, maybe move up `bool fSuccess = true;` and break in the loop? Same below in `Backup()`?